docs/tutorials/notebooks/tracking_progress.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>Please add link to Sinkhorn class.</p><p>Nice to have, but not necessary right now: can also adding link to the {class}<code>~tqdm.tqdm</code>,but this would require also adding it to the <code>intersphinx_mapping</code> in <code>conf.py</code> - will leave this up to you.</p><p>You can also consider decreasing the threshold - this will give you more iterations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Unfortunately, I could not find a valid intersphinx mapping (we tried last time together but could not find a proper one).</p>